### DISCUSSION TITLE
"Ничего не делай" внутри сценария

### DISCUSSION THREAD

**Discussion by Nik768** [dext0r/ha-yandex-station-intents#48] [Q&A] [answered]
Добрый день, благодарю за полезную интеграцию, но появился вопрос.
Интеграция создает внутри УДЯ сценарии которые выполняю команду "Сделай громкость" + знаки препинания. Однако в этом случае колонка с экраном начинает отображать текущую громкость, что как бы мелочь, но для перфекционистов больно)
Почему не используется команда "Ничего не  делай"? Я попробовал вручную на одном из сценариев заменить текст команды и все работает отлично (экран гаснет на время выполнения команды), проблема в том, что муторно так все сценарии править, они потом еще могут затереться.
Возможно с этой командой есть какие-то проблемы? Интересуюсь чисто из любопытства.

**Comment by dext0r** [comment, marked as answer]
"ничего не делай" была обнаружена недавно, если она работает - это супер. Я проверю на своих колонках и внесу изменения в компонент.
Можете вручную измененить фразу в `STATION_STUB_COMMAND` в файле `custom_components/yandex_station_intents/const.py`, перезагрузить HA, дождаться обновления сценариев, и проверить работоспособность на ваших устройствах.

**Reply by Nik768** [reply]
Поменял, спасибо, вроде все хорошо работает!

**Comment by dext0r** [comment]
https://github.com/dext0r/ha-yandex-station-intents/releases/tag/v0.6.2
